starport-blog/tutorial/04-http-create-post.md

```diff
@@ -134,6 +134,7 @@ Next, we need to define the `createPost` function. It will take the value from t
 const createPost = () => {
   // Getting a post title value from text input
   const title = document.getElementById("input").value;
+  const body = document.getElementById("body").value;
```

Comment:
the title should also be modified to `getElementById("title")`

Reply:
Thank you, @xiaojian-zhou. I'll do that.

Reply:
@xiaojian-zhou I've updated the PR with all the changes you asked. Can you please review that?